logstash-core/src/test/java/org/logstash/jackson/StreamReadConstraintsUtilTest.java

```diff
@@ -23,13 +27,12 @@ public class StreamReadConstraintsUtilTest {
     @ClassRule
     public static final LoggerContextRule LOGGER_CONTEXT_RULE = new LoggerContextRule("log4j2-log-stream-read-constraints.xml");
 
+    @Rule
+    public final DefaultsRestorer defaultsRestorer = new DefaultsRestorer();
+
     private ListAppender listAppender;
     private Logger observedLogger;
 
-    private static final int DEFAULT_MAX_STRING_LENGTH = 200_000_000;
-    private static final int DEFAULT_MAX_NUMBER_LENGTH = 10_000;
-    private static final int DEFAULT_MAX_NESTING_DEPTH = 1_000;
-
     @Before
     public void setUpLoggingListAppender() {
         int i = 1+16;
```

Comment:
```suggestion
```
I realize this is not part of the diff, but i just noticed it peaking out 😅

Reply:
Eep. Whoops. I'll follow up.

Reply:
follow-up in https://github.com/elastic/logstash/pull/16884